docker/README.md

```diff
@@ -0,0 +1,65 @@
+# Docker images of HeAT
+
+There is some flexibility to building the Docker images of HeAT.
+
+Firstly, one can build from the released version taken from PyPI. This will either be
+the latest release or the version set through the `--build-arg=HEAT_VERSION=1.2.0`
+argument.
+
+Secondly one can build a docker image from the GitHub sources, selected through
+`--build-arg=INSTALL_TYPE=source`. The default branch to be built is main, other
+branches can be specified using `--build-arg=HEAT_BRANCH=branchname`.
+
+## General build
+
+The [Dockerfile](./Dockerfile) guiding the build of the Docker image is located in this
+directory. It is typically most convenient to `cd` over here and run the Docker build as:
+
+	$ docker build .
+
+The resulting image (ID) should then be tagged for subsequent upload (push) to a
+repository, for example:
+
+	$ docker tag ea0a1040bf8a ghcr.io/helmholtz-analytics/heat:1.2.0_torch1.11_cuda11.5_py3.9
+	$ docker push ghcr.io/helmholtz-analytics/heat:1.2.0_torch1.11_cuda11.5_py3.9
+
+Please ensure that you push the same tag that you just created.
```

Comment:
Is this for the devs or the users? I'm not sure what the use case is.

Reply:
For me it is also unclear whether I have to run this when I just want to use Heat in containerized version or whether this just explains what had to be done in order to create the container.